components/MW/metgene_rxn_table/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { MetaNode } from '@/spec/metanode'
+import { z } from 'zod'
+
+// A unique name for your data type is used here
+export const MetGeneRxnTable = MetaNode.createData('MetGeneRxnTable')
+  // Human readble descriptors about this node should go here
+  .meta({
+    label: 'MetGene Reaction Table',
+    description: 'MetGene Reaction Table, rendered',
+  })
+  // this should have a codec which can encode or decode the data type represented by this node
+  //  using zod, a compile-time and runtime type-safe codec can be constructed
+  .codec(z.any())
```

Comment:
Please add schema for your type

Reply:
How do I add schema for type MetGeneRxnTable ? I need help in fixing this.